src/os/bluestore/NVMEDevice.cc

```diff
@@ -138,8 +142,14 @@ class SharedDriverQueueData {
         flush_lock("NVMEDevice::flush_lock"),
         flush_waiters(0),
         completed_op_seq(0), queue_op_seq(0) {
+    struct spdk_nvme_io_qpair_opts opts = {};
+    spdk_nvme_ctrlr_get_default_io_qpair_opts(ctrlr, &opts, sizeof(opts));
```

Comment:
do you mean spdk userspace nvme driver? If yes, isn't it supply a function to get the size driver supported?